modules/nf-core/kma/kma/tests/main.nf.test

```diff
@@ -0,0 +1,84 @@
+nextflow_process {
+
+    name "Test Process KMA_KMA"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "kma"
+    tag "kma/kma"
+
+    script "../main.nf"
+    process "KMA_KMA"
+
+    setup {
+        run("KMA_INDEX") {
+            script "../../index/main.nf"
```

Comment:
why the module structure is "modules/nf-core/kma/kma"? Shouldn't it be "modules/nf-core/kma"?

Reply:
This is due to the way kma is structured, its a bit unusual compared to common bioinformatics tools. E.g. bwa calls its submodules via the first positional argument, like `bwa index`, `bwa aln` or `bwa mem`. However, kma has a separate binary for every submodule, e.g. `kma_index` and just `kma` for the actual alignment. Hence, I separated the functionality in into subfolders of `kma/` where I can call the alignment nothing else but `kma` again (as the name of the binary).